src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -0,0 +1,110 @@
+package mate.academy.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Book;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class BookDaoImpl implements BookDao {
+    private static final String CREATE_QUERY = "INSERT INTO books (title, price) VALUES (?, ?)";
+    private static final String FIND_BY_ID_QUERY = "SELECT * FROM books WHERE id = ?";
+    private static final String FIND_ALL_QUERY = "SELECT * FROM books";
+    private static final String UPDATE_QUERY = "UPDATE books SET title = ?, price = ? WHERE id = ?";
+    private static final String DELETE_QUERY = "DELETE FROM books WHERE id = ?";
```

Comment:
from checklist
![image](https://github.com/mate-academy/jv-jdbc-intro/assets/55763108/95532752-0902-4bcc-affb-45c69357cc46)

Reply:
u dont need to use constant

Reply:
@Andriy-dz actually constants don't do any harm in this case :)

the point, mentioned to checklist is relevant when the string is inlined as an argument for `prepareStatement` method, which is not the case when using constants